packages/sources/package.json

```diff
@@ -26,7 +26,6 @@
     },
     "homepage": "https://github.com/RedHatInsights/frontend-components/tree/master/packages/sources#readme",
     "dependencies": {
-        "@redhat-cloud-services/frontend-components": "*",
```

Comment:
Is this package not using frontend components at all? I think you should also run install in the root to update the lock file.

Reply:
@Hyperkid123  there is no change to package-lock.